.github/workflows/basic.yml

```diff
@@ -7,6 +7,70 @@ env:
   INSTL_DIR : "${{github.workspace}}/build/install-dir"
 
 jobs:
+  icx-build:
```

Comment:
I think it would be better to add icx/icpx as a "include" to ubuntu matrix (similar to clang or gcc-7) if this is possible

Reply:
It can extend execution time a little bit, but I can do it on another PR.